mentors/Makefile

```diff
@@ -57,22 +57,29 @@ shell: docker-image-exists $(WATSON_CREDENTIALS)
 			-e WATSON_PASSWORD=$(WATSON_PASSWORD) \
 			--entrypoint /bin/bash \
 			-v $(shell pwd)/$(MENTOR):/app/mounts/$(MENTOR) \
+			-v $(shell pwd)/data:/app/mounts/data \
 			-v $(shell pwd)/src:/app/src \
 		$(DOCKER_IMAGE)
 
 
 # Pre-processes mentor data inside docker container
 # Generates audiochunks and transcripts for one mentor
 %/build: $(WATSON_CREDENTIALS)
+	$(MAKE) $*/build/update
+
+# Pre-processes mentor data inside docker container
+# Generates audiochunks and transcripts for one mentor
+.PHONY: %/build/update
+%/build/update: $(WATSON_CREDENTIALS)
 	@echo "creating $*/build ..."
-	$(MAKE) docker-image-exists 
+	$(MAKE) docker-image-exists
```

Comment:
this can be a regular dependency. we only did it differently in the `data` rule to avoid unwanted rerun of the rule when data exists

```
%/build/update: docker-image-exists $(WATSON_CREDENTIALS)
```